composer.json

```diff
@@ -69,7 +69,8 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.0.x-dev"
+            "dev-master": "2.0.x-dev",
+            "dev-fix-undefined-index": "2.0.x-dev"
```

Comment:
This doubles the alias, it only leads to errors, please revert it.

Reply:
Fixed